templates/main.html

```diff
@@ -4,10 +4,18 @@
     <meta charset="utf-8">
     <title>GCI Leaders</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta property="og:title" content="GCI Current Leaders">
+    <meta property="og:description" content="Google Code-in 2017 Current Leaders">
+    <meta property="og:image" content="{{rootURL}}/static/images/GCI-logo.jpg">
```

Comment:
The value is being escaped by Mustache

Use triple bracket `{{{rootURL}}}` to prevent that.